src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="position">
```

Comment:
Use semantic tags where possible. The class name 'position' does not indicate the purpose of the <body> content. Consider using a more descriptive class name or semantic HTML5 elements to improve SEO and accessibility.